frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/EnvironmentAccordionBody/StrategyDraggableItem/StrategyItem/CopyStrategyIconMenu/CopyStrategyIconMenu.tsx

```diff
@@ -179,7 +179,10 @@ export const CopyStrategyIconMenu: VFC<ICopyStrategyIconMenuProps> = ({
                                         }
                                     />
                                     <ListItemText>
-                                        Copy to {environment}
+                                        Copy to{' '}
+                                        {environment === environmentId
+                                            ? 'current'
+                                            : environment}
```

Comment:
There's an edge case, here, right? 
- If you have a different environment named "current", you'll have two "Copy to current" options.

I don't think it's very important ("current" is a weird name for an env, anyway), but it might be nice to consider if we can distinguish this option from the other options somehow 🤷🏼

Reply:
really good catch. I was also considering "Duplicate in current" to make it stand out from the list

Reply:
<img width="306" alt="Screenshot 2024-08-02 at 09 24 18" src="https://github.com/user-attachments/assets/cc297184-a806-43c8-8e41-5865182efd61">